Include/XA/czyz/cyyz_stoplines.geo

```diff
@@ -0,0 +1,145 @@
+N043.40.15.864;W079.36.59.181;N043.40.15.152;W079.36.58.290;STOPBAR;
+//Untitled Path
+N043.40.13.733;W079.36.56.576;N043.40.12.766;W079.36.55.115;STOPBAR;
+//Untitled Path
+N043.40.20.705;W079.36.53.045;N043.40.19.910;W079.36.51.827;STOPBAR;
+//Untitled Path
+N043.40.18.750;W079.36.50.522;N043.40.17.573;W079.36.48.693;STOPBAR;
+//Untitled Path
+N043.39.41.472;W079.37.27.706;N043.39.44.231;W079.37.23.677;STOPBAR;
+//Untitled Path
+N043.39.54.032;W079.37.11.124;N043.39.55.345;W079.37.11.524;STOPBAR;
+//Untitled Path
+N043.39.55.811;W079.37.10.262;N043.39.56.974;W079.37.08.556;STOPBAR;
+//Untitled Path
+N043.40.05.509;W079.36.52.894;N043.40.06.581;W079.36.51.302;STOPBAR;
+//Untitled Path
+N043.40.07.683;W079.36.49.217;N043.40.09.216;W079.36.49.686;STOPBAR;
+//Untitled Path
+N043.40.16.647;W079.36.40.386;N043.40.17.840;W079.36.38.636;STOPBAR;
+N043.40.17.840;W079.36.38.636;N043.40.18.044;W079.36.36.967;STOPBAR;
+//Untitled Path
+N043.40.24.559;W079.36.26.716;N043.40.25.631;W079.36.25.132;STOPBAR;
+//Untitled Path
+N043.40.32.593;W079.36.16.283;N043.40.33.055;W079.36.14.581;STOPBAR;
+N043.40.33.055;W079.36.14.581;N043.40.32.844;W079.36.13.588;STOPBAR;
+//Untitled Path
+N043.40.36.756;W079.36.08.947;N043.40.37.862;W079.36.07.353;STOPBAR;
+//Untitled Path
+N043.40.45.264;W079.35.55.136;N043.40.47.986;W079.35.51.193;STOPBAR;
+//Untitled Path
+N043.40.41.547;W079.35.47.611;N043.40.40.981;W079.35.45.669;STOPBAR;
+//Untitled Path
+N043.40.36.459;W079.35.50.066;N043.40.35.971;W079.35.48.352;STOPBAR;
+//Untitled Path
+N043.40.31.348;W079.36.01.972;N043.40.32.350;W079.36.00.474;STOPBAR;
+//Untitled Path
+N043.40.28.292;W079.36.02.146;N043.40.27.767;W079.36.00.306;STOPBAR;
+//Untitled Path
+N043.40.19.141;W079.36.19.689;N043.40.20.071;W079.36.18.268;STOPBAR;
+//Untitled Path
+N043.40.15.959;W079.36.19.800;N043.40.15.480;W079.36.18.166;STOPBAR;
+//Untitled Path
+N043.40.10.578;W079.36.32.163;N043.40.11.508;W079.36.30.743;STOPBAR;
+//Untitled Path
+N043.40.07.386;W079.36.32.273;N043.40.06.915;W079.36.30.604;STOPBAR;
+//Untitled Path
+N043.40.00.875;W079.36.46.444;N043.40.01.836;W079.36.44.976;STOPBAR;
+//Untitled Path
+N043.40.01.681;W079.36.40.571;N043.40.00.466;W079.36.39.973;STOPBAR;
+//Untitled Path
+N043.39.49.817;W079.37.02.503;N043.39.50.849;W079.37.00.970;STOPBAR;
+//Untitled Path
+N043.39.49.329;W079.36.56.061;N043.39.50.742;W079.36.56.734;STOPBAR;
+//Untitled Path
+N043.39.34.758;W079.37.17.460;N043.39.36.310;W079.37.17.347;STOPBAR;
+//Untitled Path
+N043.39.31.901;W079.37.21.951;N043.39.32.867;W079.37.20.521;STOPBAR;
+//Untitled Path
+N043.39.35.421;W079.37.23.652;N043.39.35.402;W079.37.21.535;STOPBAR;
+//Untitled Path
+N043.39.36.124;W079.37.20.720;N043.39.37.088;W079.37.19.321;STOPBAR;
+//Untitled Path
+N043.40.45.044;W079.37.26.629;N043.40.46.253;W079.37.26.152;STOPBAR;
+//Untitled Path
+N043.41.11.259;W079.38.00.851;N043.41.11.580;W079.37.58.812;STOPBAR;
+//Untitled Path
+N043.40.55.203;W079.37.39.458;N043.40.55.400;W079.37.38.011;STOPBAR;
```

Comment:
Fine for now, but next time there's an update it'd be nice to find/replace `//Untitled Path` out so it's not cluttering up the file.